exporter.py

```diff
@@ -154,15 +156,25 @@ def __init__(self):
                       self.bbs[len(self.bbs) - 1]['start_time']).total_seconds()
                      if self.bbs else 0)
         )
+        self.last_backup_size = Gauge('walg_last_backup_size',
+                                 'Size of last uploaded backup. Label compressed=yes for  compressed size and no for uncompressed ',
```

Comment:
https://prometheus.io/docs/practices/naming/#labels
Use labels to differentiate the characteristics of the thing that is being measured:
Do not put the label names in the metric name, as this introduces redundancy and will cause confusion if the respective labels are aggregated away.

I think a `compression` label with value `compressed` or `uncompressed` would make more sense here.